assets/dev/scss/frontend/_widgets.scss

```diff
@@ -24,3 +24,9 @@
 @import "widgets/toggle";
 @import "widgets/video";
 @import "widgets/rating";
+
+//
+// Modules with Widgets
+//
+@import '../../../../modules/nested-tabs/assets/scss/frontend';
+@import '../../../../modules/nested-accordion/assets/scss/frontend';
```

Comment:
@louiswol94 Maybe it should be placed at the top of the file (line 1), above the widget, to keep the same loading order? For BC.

Reply:
@rami-elementor Sure, updated!